src/website/views/components/marketing-icon/_examples.pug

```diff
@@ -0,0 +1,104 @@
+h2 Overview
+p.-text
+  | Chi Marketing Icons are a collection of high-quality vector icons designed by Lumen for use in consumer, enterprise, and internal applications.
+  | All icons have been manually optimized to achieve the fastest load time without compromising visual path and shape quality. For icon requests or
+  | feedback, please email <a href="mailto:chi.support@centurylink.com">Chi Support</a>.
+
+h2 Examples
+h3 Base
+.example.-mb--3
+  .-p--3
+    chi-marketing-icon(icon="business-big-data")
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-default
+      li.-active
+        a(href='#webcomponent-default') Web Component
+      li.-disabled
+        a(href='#html-default') HTML Blueprint
```

Comment:
```suggestion
        a(href='#html-default', role='tab', aria-selected='false', tabindex='-1', aria-controls='html-default') HTML Blueprint
```

Should we add accessibility attributes to the new tabs that we add to the documentation?
1. `tabindex="-1"` on disabled ones to avoid this behavior
<img width="490" alt="Screenshot 2021-03-03 at 15 33 52" src="https://user-images.githubusercontent.com/25136131/109821065-e14d9100-7c35-11eb-8065-8c51db54ad83.png">

2. `aria-controls="ID"` indicating ID of the respective tab panel

3. `aria-selected=""`

4. `role="tab"`

Reply:
Yes, good point. I've added to all tab items in the Marketing Icon doc page. Thanks!